opencti-platform/opencti-graphql/tests/02-integration/01-database/elasticSearch-test.js

```diff
@@ -419,7 +419,7 @@ describe('Elasticsearch pagination', () => {
     let data = await elPaginate(testContext, ADMIN_USER, READ_ENTITIES_INDICES, { search: 'malicious' });
     expect(data.edges.length).toEqual(21);
     data = await elPaginate(testContext, ADMIN_USER, READ_ENTITIES_INDICES, { search: 'with malicious' });
-    expect(data.edges.length).toEqual(45);
+    expect(data.edges.length).toEqual(46);
```

Comment:
Why we have an increment here ?

Reply:
46 instead of 45 because of the new role Knowledge with description 'Role with knowledge capabilities' that is know counted because it contains 'with'

Reply:
that's much clearer, thank you!